js/start.md

```diff
@@ -425,8 +425,11 @@ class App extends Component {
             const url = 'https://console.aws.amazon.com/pinpoint/home/?region=us-east-1#/apps/'+awsconfig.aws_mobile_analytics_app_id+'/analytics/events';
             let result = (<div>
               <p>Event Submitted.</p>
-              <p>Events sent: {++this.state.eventsSent}</p>
-              <a href={url} target="_blank">View Events on the Amazon Pinpoint Console</a>
+              <p>Events sent: {this.setState({ eventsSent: this.state.eventsSent + 1})}
+              </p>
+              <a href={url} target="_blank" rel="noopener noreferrer">
```

Comment:
I would revert this change to not make the docs so long

Reply:
I think docs being a little longer is more preferable than CRA's lint warnings.